OWNERS_ALIASES

```diff
@@ -2,14 +2,18 @@ aliases:
   admins:
     - justinsb
     - micahhausler
-    - nckturner
-    - wongma7
     - xdu31
     - yue9944882
+    - ganiredi
+    - mahakagi
+    - markapruett
```

Comment:
I don't think anyone here is in the kubernetes-sigs GH org yet, so I don't know that this will have any effect.

Reply:
right, need to add you to k-sigs orgs before adding to the OWNER list